src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library_view.cljs

```diff
@@ -11,22 +11,25 @@
 
 
 (defn- render-value [value]
+  ;; figure out what to do with typeahead?
   (cond (sequential? value) (clojure.string/join ", " value)
         (common/attribute-list? value) (clojure.string/join ", " (common/attribute-values value))
         :else value))
 
-(defn- render-library-row [key value]
+(defn- render-library-row [key value doid] ;; doid
   (when value
     [:div {:style {:display "flex" :padding "0.5em 0" :borderBottom (str "2px solid " (:line-default style/colors))}}
      [:div {:style {:flexBasis "33%" :fontWeight "bold" :paddingRight "2em"}} key]
-     [:div {:style {:flexBasis "67%"}} value]]))
+     [:div {:style {:flexBasis "67%"}} value
+      (if (not (nil? doid)) [:span {:style {:fontStyle "italic"}} (str " (" doid ")")])]]))
 
 
 (defn- render-property [library-schema library-attributes property-key]
   (render-library-row
    (get-in library-schema [:properties property-key :title])
-   (render-value 
-    (or (get library-attributes property-key) (get-in library-schema [:properties property-key :default])))))
+   (render-value
+    (or (get library-attributes property-key) (get-in library-schema [:properties property-key :default])))
+    (if (= (get-in library-schema [:properties property-key :typeahead]) "ontology") (get library-attributes :library:diseaseOntologyID) nil)))
```

Comment:
to avoid hardcoding, how about something like (pseudocode):
```
if (typeahead === "ontology") {
  replace a "Label" suffix with an "ID" suffix on the property-key and display the ID
}
```